### DISCUSSION TITLE
FO: Add language selection at maintenance page

### DISCUSSION THREAD

**Discussion by abhishek-webkul** [PrestaShop/PrestaShop#33137] [Ideas]
### The problem

The visitors are unable to get the maintenance message in their language of comfort. A language selection option as available on all other pages will suffice. 

### The solution

To add language selector on maintenance page at front office.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
Hello @abhishek-webkul 

It is a really interesting idea. I'll ping the product team, so they can validate it, especially since you are motivated to help the project with Pull Request 👍🏻 

One question from me. How would you like this to be implemented? ~~All text available in the markup, and some javascript switch to change the language?~~

**Comment by MatShir** [comment]
Cool idea ! Don't hesitate to join the [slack](https://join.slack.com/t/prestashop/shared_invite/zt-1xe2sd7he-zTJheIinZkujzrox68_X2g) for any questions during your contribution 🤗

**Comment by eternoendless** [comment]
The language could be auto-detected using the browser's `Accept-Language` request header.

**Reply by kpodemski** [reply]
It's already the case. I think the idea is to have it easily switchable
